eslzArm/eslzArm.terraform-sync.param.json

```diff
@@ -1,620 +1,878 @@
+      "value": "Audit"
+    },
+    "enableWsMySQLInitiatives": {
+      "value": "Audit"
+    },
+    "enableWsNetworkInitiatives": {
+      "value": "Audit"
+    },
+    "enableWsOpenAIInitiatives": {
+      "value": "Audit"
+    },
+    "enableWsPostgreSQLInitiatives": {
+      "value": "Audit"
+    },
+    "enableWsServiceBusInitiatives": {
+      "value": "Audit"
+    },
+    "enableWsSQLInitiatives": {
+      "value": "Audit"
+    },
+    "enableWsStorageInitiatives": {
+      "value": "Audit"
+    },
+    "enableWsSynapseInitiatives": {
+      "value": "Audit"
+    },
+    "enableWsVirtualDesktopInitiatives": {
+      "value": "Audit"
+    },
+    "enforceAcsb": {
+      "value": "Yes"
+    },
+    "enforceBackup": {
+      "value": "Yes"
+    },
+    "enforceBackupPlat": {
+      "value": "Yes"
+    },
+    "enforceKvGuardrails": {
+      "value": "Yes"
+    },
+    "enforceKvGuardrailsPlat": {
+      "value": "Yes"
+    },
+    "enterpriseScaleCompanyPrefix": {
+      "value": "defaults"
+    },
+    "erAzSku": {
+      "value": ""
+    },
+    "erAzSkuSecondary": {
+      "value": ""
+    },
+    "erRegionalOrAz": {
+      "value": ""
+    },
+    "erRegionalOrAzSecondary": {
+      "value": ""
+    },
+    "erRegionalSku": {
+      "value": ""
+    },
+    "erRegionalSkuSecondary": {
+      "value": ""
+    },
+    "expressRouteScaleUnit": {
+      "value": "1"
+    },
+    "expressRouteScaleUnitSecondary": {
+      "value": "1"
+    },
+    "firewallSku": {
+      "value": "Premium"
+    },
+    "firewallSkuSecondary": {
+      "value": "Premium"
+    },
+    "firewallZones": {
+      "value": [
+        "1",
+        "2",
+        "3"
+      ]
+    },
+    "firewallZonesSecondary": {
+      "value": [
+        "1",
+        "2",
+        "3"
+      ]
+    },
+    "gwAzSku": {
+      "value": ""
+    },
+    "gwAzSkuSecondary": {
+      "value": ""
+    },
+    "gwRegionalOrAz": {
+      "value": ""
+    },
+    "gwRegionalOrAzSecondary": {
+      "value": ""
+    },
+    "gwRegionalSku": {
+      "value": ""
+    },
+    "gwRegionalSkuSecondary": {
+      "value": ""
+    },
+    "identityAddressPrefix": {
+      "value": "10.110.0.0/24"
+    },
+    "identityAddressPrefixSecondary": {
+      "value": "10.210.0.0/24"
+    },
+    "identitySubscriptionId": {
+      "value": "00000000-0000-0000-0000-000000000000"
+    },
+    "internetTrafficRoutingPolicy": {
+      "value": false
+    },
+    "internetTrafficRoutingPolicySecondary": {
+      "value": false
+    },
+    "laCategory": {
+      "value": "allLogs"
+    },
+    "listOfResourceTypesDisallowedForDeletion": {
+      "value": [
+        "microsoft.managedidentity/userassignedidentities"
+      ]
+    },
+    "managementSubscriptionId": {
+      "value": "00000000-0000-0000-0000-000000000000"
+    },
+    "monitorAlertsResourceGroup": {
+      "value": "rg-amba-monitoring-001"
+    },
+    "onlineLzSubscriptionId": {
+      "value": []
+    },
+    "privateDnsZonesToDeploy": {
```

Comment:
Seems to be some non-relevant bits like this. I guess it doesn't matter, but flagging just in case this wasn't intentional.